server/rollback.go

```diff
@@ -40,16 +40,38 @@ application.
 				ctx.Viper,
 			)
 
+			// App State
+			lastCommit := app.CommitMultiStore().LastCommitID()
+			fmt.Printf("Initial App state height=%d and hash=%X\n", lastCommit.GetVersion(), lastCommit.GetHash())
```

Comment:
Adding more print statements, when rolling back we should be a bit more verbose to understand the before an after pictures